src/Features/CSharp/Portable/DuplicateMethod/CSharpDuplicateMethodCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Composition;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.DuplicateMethod;
+
+namespace Microsoft.CodeAnalysis.CSharp.DuplicateMethod
```

Comment:
The main use case *for you* :)

But if we're adding this as a full fledged IDE feature, i'd like it to be a little more broad applicable.   Say someone way making a bunch of properties that all followed a similar pattern.  It would be nice to have this simple 'copy/paste'